doc/src/guides/solving/solve-ode.md

```diff
@@ -0,0 +1,423 @@
+(solving-guide-inequalities)=
+# Solve an Ordinary Differential Equation (ODE) Algebraically
+
+Use SymPy to solve an ordinary differential equation (ODE) algebraically. For
+example, solving $y''(x) + 9y(x)=0 $ yields $ y(x)=C_{1} \sin(3x)+ C_{2}
+\cos(3x)$.
+
+## Alternatives to Consider
+- To numerically solve a system of ODEs, use a [SciPy ODE
+  solver](https://docs.scipy.org/doc/scipy/reference/integrate.html#solving-initial-value-problems-for-ode-systems)
+  such as `solve_ivp`. You can also use SymPy to create and then
+  {func}`~.lambdify` an ODE to be solved numerically using SciPy's as
+  `solve_ivp` as described below in [](#numerically-solve-an-ode-in-scipy).
+
+## Solve an Ordinary Differential Equation (ODE)
+
+Here is an example of solving an ordinary differential equation algebraically
+using {func}`~.dsolve`:
+
+```py
+>>> from sympy import Function, dsolve, Derivative
+>>> from sympy.abc import x
+>>> y = Function('y')
+>>> result = dsolve(Derivative(y(x), x, x) + 9*y(x), y(x))
+>>> result
+Eq(y(x), C1*sin(3*x) + C2*cos(3*x))
+```
+
+You can then use SymPy to verify that the solution is correct:
+
+```py
+>>> from sympy import checkodesol
+>>> solution = result.rhs
```

Comment:
It's not necessary to do this. `checkodesol` can accept the direct Eq output of `dsolve`.